### DISCUSSION TITLE
[General] Topic 이름에 대한 convention은 어떤 것이 있을까?

### DISCUSSION THREAD

**Discussion by kyungjunleeme** [kyungjunleeme/kafka_study#18] [Q&A]
cf) https://kafka.apache.org/documentation/#multitenancy-topic-naming

**Comment by chaeyeonhan1225** [comment]
`product-created`, `content-saved` 
저의 경험으로는 행위를 과거형으로 표현한 이벤트 명을 주로 사용했습니다.
https://stackoverflow.com/questions/12510535/naming-convention-for-commands-events

**Reply by kyungjunleeme** [reply]
오호 그렇군요. 근데 카프카 한정은 아니라 약간 범위가 넓은영역이긴하네요

**Comment by kyungjunleeme** [comment]
https://flavono123.oopy.io/posts/kafka-topic-naming-convention-story

여기에 잘 나와있음.
